src/importer/ScoreLoader.ts

```diff
@@ -76,8 +76,7 @@ export class ScoreLoader {
         let importers: ScoreImporter[] = Environment.buildImporters();
         Logger.debug(
             'ScoreLoader',
-            'Loading score from ' + data.length + ' bytes using ' + importers.length + ' importers',
-            null
+            'Loading score from ' + data.length + ' bytes using ' + importers.length + ' importers'
```

Comment:
As we are touching the code here, might be good to directly convert it to template strings.